bin/git/squash-commits.sh

```diff
@@ -81,7 +81,7 @@ owner=$(gh pr view --json "headRepositoryOwner" --jq '.headRepositoryOwner.login
 user=$(gh api user --jq '.login')
 echo "The PR is owned by '${owner}'. The current user is '${user}'."
 if [ "${owner}" = 'XRPLF' ] || [ "${owner}" = "${user}" ]; then
-  remote="origin"
+  remote="$(git remote -v | grep 'XRPLF/rippled.git (push)' | cut -f1)"
```

Comment:
```suggestion
  remote="$(git remote -v | grep ':XRPLF/rippled.git (push)' | head -1 | cut -f1)"
```
1. Note the added `:`. That will ensure only SSH URIs are included, and specifically exclude `https` Github URIs. For example, I have my `upstream` remote configured as `https://github.com/XRPLF/rippled.git` to prevent myself from _accidentally_ pushing to it.
2. The added `head -1` allows this to work if the user has more than one remote defined with the same URI, which is also something I do on my end.

Reply:
Done.